### ISSUE TITLE
[pypi] nitrokeyapp package lacks compiled *.ui resources

### ISSUE THREAD

**Issue by brevilo** [Nitrokey/nitrokey-app2#162]
This might be a stupid question, but what's the current purpose of the [`nitrokeyapp`](https://pypi.org/project/nitrokeyapp/2.1.3/) package on PyPi.org? My assumption has been that installing the package will allow to directly run the app.

However,  the `nitrokeyapp/ui/*.ui` files apparently aren't compiled to their `*.py` counterparts (as the [`Makefile`](https://github.com/Nitrokey/nitrokey-app2/blob/d0e682a197aee5a900bb33ec5cbe08fee7a76a32/Makefile#L36) suggests), so launching `nitrokeyapp` (on macOS) will inadvertently result in:
```
  File "[...]/lib/python3.11/site-packages/nitrokeyapp/error_dialog.py", line 9, in <module>
    from nitrokeyapp.ui.error_dialog import Ui_ErrorDialog
ModuleNotFoundError: No module named 'nitrokeyapp.ui.error_dialog'
```

Am I missing something?

Thanks

**Comment by brevilo** [comment]
Great! That was fast! 👍
Would be awesome if you could push a fresh release to PyPi as well such that the package becomes usable.

Keep up the nice work!

**Comment by mmerklinger** [comment]
We are planning a release in the next few days. Currently we're considering to include a few more smaller fixes. In the meantime you could also compile the wheel locally, with the `build` target of the makefile. Please note that macOS is currently not officially supported, but we're working on this as well.

**Comment by brevilo** [comment]
Sorry @mmerklinger, I need to reopen this issue as it's not yet resolved. a991314 is included in the 2.1.4 release but the PyPI [wheel](https://files.pythonhosted.org/packages/6f/0a/8f11709df01973cd1037b55d381926c366feec6399f7f20bdd07e5b79b9a/nitrokeyapp-2.1.4-py3-none-any.whl) doesn't yet include the missing the `nitrokeyapp/ui/*.py` files. Neither does the [source archive](https://files.pythonhosted.org/packages/dd/16/9d9e6f4d2d5a08b32c14de9a21dcf191fb5e9e4f597e5b441793d2f0d9ae/nitrokeyapp-2.1.4.tar.gz).

**Comment by mmerklinger** [comment]
Yes, you are totally right. Sorry! I just realized that another issue I fixed a few days ago also had an impact on this. With my change in #163 I fixed the build job in our PyPI-Pipeline, but another issue was that the artifacts from this job weren't correctly downloaded in the subsequent publish job to PyPI. This caused *flit*, the old build tool, to rebuild the wheel without the files included. We currently do the migration to Qt6 and then will release a new version.
